glnx-fdio.c

```diff
@@ -387,6 +387,29 @@ glnx_link_tmpfile_at (GLnxTmpfile *tmpf,
   return TRUE;
 }
 
+/**
+ * glnx_openat_read:
+ * @dfd: File descriptor for origin directory
+ * @path: Pathname, relative to @dfd
+ * @error: Error
+ *
+ * Use openat() to open a file, with flags `O_RDONLY | O_CLOEXEC | O_NOCTTY`.
+ * Like the other libglnx wrappers, will use `TEMP_FAILURE_RETRY` and
+ * also includes @path in @error in case of failure.
+ */
+gboolean
+glnx_openat_read (int             dfd,
```

Comment:
Can we call it e.g. `glnx_openat_rdonly` ? `openat_read` makes me think it actually reads the file as well.